benchmark_aux.c

```diff
@@ -121,3 +124,28 @@ uint32_t get_cores(void)
 {
 	return sysconf(_SC_NPROCESSORS_ONLN);
 }
+
+void *mmap_paddr(intptr_t paddr, uint64_t size)
+{
+	int   devmem;
```

Comment:
nit: Single space between int and devmem.

Reply:
fixed